### PR TITLE
CORE-450 sundry-type endpoint config

### PR DESCRIPTION
**Pull request by orzeuek**

### CODE REVIEW

File: config/common.ts [hunk 1]
```diff
@@ -28,6 +28,10 @@ export default class Config {
             name: 'ride-service',
             cacheConfig: {maxAge: DEFAULT_MAX_AGE, max: 4000},
         },
+        '^\/sundry-type\/[0-9a-zA-Z]+': {
+            name: 'sundry-service',
+            cacheConfig: {maxAge: DEFAULT_MAX_AGE, max: 4000},
```
Comment (linusnorton) [inline config/common.ts]: max should be about 20

Comment (orzeuek) [inline config/common.ts]: 20 seconds cache for hardcoded map of values ..... are you sure?

Comment (linusnorton) [inline config/common.ts]: maxAge is the max amount of time, max is the maximum number of items

Comment (orzeuek) [inline config/common.ts]: point for you :)

**Pull request merged**